synapse/lib/view.py

```diff
@@ -1086,7 +1098,7 @@ async def runStorm():
                     tock = tick + abstook
                     await chan.put(('fini', {'tock': tock, 'abstock': abstock, 'took': abstook, 'count': count, }))
 
-        await synt.worker(runStorm(), name='runstorm')
+        self.core.boss.schedCoro(runStorm())
```

Comment:
This creates a task that will never be awaited; thus we'll never do cancellation cleanup of running tasks. That could present as a memory leak :chart_with_upwards_trend: